test/scripts/start-services.sh

```diff
@@ -76,10 +75,11 @@ start_polkadot_launch()
     echo "Writing Polkadot configuration"
     polkadotbinary=/tmp/polkadot/target/release/polkadot
     if [[ -f ../test/.env ]]; then
+        # shellcheck disable=SC1091
         source ../test/.env
     fi
 
-    if [ $# -eq 1 ] && [ $1 = "duplicate" ];
+    if [ $# -eq 1 ] && [ "$1" = "duplicate" ];
```

Comment:
does this still work? did u test?

Reply:
The only effect of this change is that if you call `start-services.sh "foo bar"`, then it will compare "foo bar" to "duplicate", instead of "foo". This is more correct behavior. Basically individual command line args should be quoted in scripts as a good practice.

Reply:
anyway, I've fixed the duplicate thingy and it all works now.